presto-main/src/test/java/io/prestosql/execution/buffer/BenchmarkBlockSerde.java

```diff
@@ -384,10 +385,13 @@ public void setup()
     public static class LongDecimalWithNullBenchmarkData
             extends LongDecimalBenchmarkData
     {
+        @Param({".01", ".10", ".50", ".90", ".99"})
```

Comment:
Could we extract data generation, so that new way of data gen is also used in `io.prestosql.orc.BenchmarkColumnReaders` (and potentially Parquet too)?

Reply:
Those params cannot unfortunately be extracted. So every case needs to have this field.

Reply:
> Those params cannot unfortunately be extracted. So every case needs to have this field.

These params could be part of `io.prestosql.execution.buffer.BenchmarkBlockSerde.BenchmarkData` (then they need to be defined once)
`io.prestosql.execution.buffer.BenchmarkBlockSerde.BenchmarkData` could accept some data generator.
This would the composition that @kokosing mentioned.

Reply:
I am going to see if JMH will work with that.